### PR TITLE
CORTX-31968 - Parametrized repo_health build

### PR DESCRIPTION
**Pull request by shailesh-vaidya**
# Problem Statement
- Repo health pipeline was hardcoded to use `main` branch and `https://github.com/Seagate/cortx` repo url. Modified it to take it as a parameter. 

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM
- [x] Jenkins pipelines used for testing - https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/Reports/job/repo-health/13/ 

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: scripts/reports/repo-health.sh [hunk 1]
```diff
@@ -22,6 +22,8 @@ set -eo pipefail
 
 if [ -z "$GH_OATH" ]; then echo "GH_OATH is not set. Existing..."; exit 1; fi
 if [ -z "$CODACY_OATH" ]; then echo "CODACY_OATH is not set. Existing..."; exit 1; fi
+if [ -z "$CORTX_REPO" ]; then echo "CODACY_OATH is not set. Existing..."; exit 1; fi
```
Comment (gauravchaudhari02) [inline scripts/reports/repo-health.sh:25]: change `CODACY_OATH` to var in error message.

File: scripts/reports/repo-health.sh [hunk 2]
```diff
@@ -22,6 +22,8 @@ set -eo pipefail
...
+if [ -z "$CORTX_BRANCH" ]; then echo "CODACY_OATH is not set. Existing..."; exit 1; fi
```
Comment (gauravchaudhari02) [inline scripts/reports/repo-health.sh:26]: change `CODACY_OATH` to var in error message.

File: scripts/reports/repo-health.sh (hunk 1 above)
Comment (shailesh-vaidya) [reply scripts/reports/repo-health.sh]: Fixed. Please review

File: scripts/reports/repo-health.sh (hunk 2 above)
Comment (shailesh-vaidya) [reply scripts/reports/repo-health.sh]: Fixed. Please review

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**